generators/generator-constants.js

```diff
@@ -47,9 +47,6 @@ const JACOCO_VERSION = '0.8.6';
 const KAFKA_VERSION = '5.5.2';
 const JACKSON_DATABIND_NULLABLE_VERSION = '0.2.1';
 
-// NPM packages version
-const HUSKY_VERSION = '4.2.5';
```

Comment:
this is needed by `package.json` in server/ folder, that's why the build failed I think

Reply:
Thanks for the hint, totally right.